Apps/WebClient/src/ClientApp/src/models/timelineEntry.ts

```diff
@@ -7,6 +7,7 @@ export enum EntryType {
     Medication = "Medication",
     Immunization = "Immunization",
     Covid19LaboratoryOrder = "Laboratory",
+    LaboratoryOrder = "Laboratory",
```

Comment:
This might cause problems, since enum values should be unique.

Reply:
@MikeLyttle  I just used the same module and put a comment on the code that Laboratory and COVID-19 shares the same module